src/bin.ts

```diff
@@ -6,9 +6,13 @@
  * @since 0.2.0
  */
 
-import chalk from 'chalk'
-
 import { main } from '.'
+import chalk from 'chalk'
 
-// tslint:disable-next-line: no-console
-main().catch((e) => console.log(chalk.bold.red(`Unexpected error: ${e}`)))
+main().catch((e) => {
```

Comment:
I think that's a good improvement.

Better not part of this PR, but maybe it's a good moment to discuss the following:

Recently I was thinking of how someone would use `docs-ts` not as a CLI but import it directly from another app,

Like:

```ts
import {main} from "docs-ts/Core"

main()
```

However, this would not contain the `catch` code, which I would consider an essential part of the program. It could make sense to keep `bin.ts` as lean as possible (just import main and execute it). And instead have `Core.ts` an export `main` of type `Effect<void>`. The original `main` would have to be renamed to `run` or the like.

But currently this is not convenient because there is no `catch : <A>(f: (nativeError: Error) => IO<A>) => IO<A>` inside `fp-ts/IO`. PureScript e.g. provides this function for it's Effect type (equvivalent to fp-ts's IO). It's part of the `MonadThrow` class, for which  [Effect has an instance](https://github.com/purescript/purescript-transformers/blob/v5.0.0/src/Control/Monad/Error/Class.purs#L83).

What do you think about it? Would it make sense to discuss this further at `fp-ts` or is there anything we can do here? In case you agree with the concern.

Reply:
Now I see, `fp-ts` even has a `MonadThrow` module, but no instance implemented in `IO` yet.

Reply:
@thought2 `IO` is supposed to never fail https://gcanti.github.io/fp-ts/modules/IO.ts.html